styles.css

```diff
@@ -1,77 +1,73 @@
 body {
   background-color: #EEEEEE;
   margin: 0;
-  width:100%
-
+  width: 100%
 }
 
 .like {
+  display: inline-block;
   height: 18px;
-  width: 20px;
   margin: auto;
-  padding-left: 20px;
-  padding-right: 20px;
-  display:inline-block;
   position: relative;
+  width: 20px;
 }
 
 .share {
+  display: inline-block;
   height: 15px;
   width: 20px;
-  padding-left: 20px;
-  padding-right: 20px;
-  display:inline-block;
 }
 
-#social {
-  position: relative;
-  left:200px;
+.social {
   display: inline-block;
+  left: 200px;
+  position: relative;
 }
 
-#video_info {
+.video_info {
   border-bottom: 1px gray solid;
-  padding:30px;
-  border-top:10px #EEEEEE
+  border-top: 10px #EEEEEE;
+  padding: 30px;
 }
 
-#channel_info{
-  display:grid;
+.channel_info {
+  display: grid;
   grid-template-columns: 3 auto;
   grid-template-rows: 2 auto;
```

Comment:
You have to put a unit after 3 and 2 (px, rm, etc.)

Reply:
I used 'px'  but I am not sure if that is the correct unit to use.

Reply:
I used 'px' but I am not sure that is the correct unit.